modules/core/src/adapter/type-utils/vertex-format-from-attribute.ts

```diff
@@ -0,0 +1,50 @@
+// luma.gl, MIT licese
+import {TypedArray} from '../..';
+import {DataType, VertexFormat} from '../types/vertex-formats';
+
+/** Get the vertex format for an attribute with TypedArray value and size*/
+export function getVertexFormatFromAttribute({value, size}: {value: TypedArray, size?: number}): VertexFormat {
```

Comment:
I would prefer positional parameters here. We are moving away from the accessor concept.